guardian/support-frontend
# Cover cost of transaction cost

## What are you doing in this PR?

Add functionality for a user to add 4% to a one time contribution to cover the cost of transaction. Implemented as a checkbox but design is TBC.



[**Trello Card**](https://trello.com/c/RshnFVy7/1032-experiment-adding-cover-transaction-costs-tickbox-to-page-2-on-one-time-journey-checkout-a-b-test)

## Why are you doing this?

A small percentage of our acquisition revenue goes towards covering the cost transaction, we are testing letting users have the option to cover the cost of transaction fees to direct more money to our mission when they choose to make a one-time contribution.



## How to test

Go to `uk/contribute/checkout?selected-contribution-type=one_off#ab-coverTransactionCost=variant`. See checkbox to add 4%. Check checkbox. Amount in CTA changes to add 4% (eg choose 100 see 104). Selected choice card remains the same. Pay. Thank you page shows amount + 4%.



## How can we measure success?

No statistically significant difference in landing page conversion rate

At least 30% opt-in for “cover cost”



## Screenshots
CTA amount updates 
<img width="442" alt="image" src="https://github.com/user-attachments/assets/6e8ebb0f-282e-4186-a6f5-83d9d8e4c579">

Thank you page correct
<img width="746" alt="image" src="https://github.com/user-attachments/assets/eefcfbbf-69a0-4bd5-893d-9538b19433ac">

Amount correct in MMA
<img width="411" alt="image" src="https://github.com/user-attachments/assets/f82e05ae-a407-4f85-bbca-db5a49bddacd">
